src/tests/super-admins.test.js

```diff
@@ -1 +1,124 @@
-test.skip('skip', () => {});
+/* eslint-disable no-underscore-dangle */
+import request from 'supertest';
+import app from '../app';
+import superAdmins from '../models/super-admins';
+import superAdminsSeed from '../seeds/super-admins';
+
+const mockSuperAdminSameInfo = {
+  firstName: 'SuperAdmin',
+  email: 'SuperAdmin@gmail.com',
+  password: 'SuperAdmin1234',
+};
+const mockSuperAdminName = {
+  firstName: 'SAdminChange',
+};
+const mockSuperAdminEmail = {
+  email: 'SAdminChange@gmail.com',
+};
+const mockSuperAdminPass = {
+  password: 'SAdminChange1234',
+};
+const mockSuperAdminEmailOnDB = {
+  email: 'SuperAdmin2@gmail.com',
+};
+const mockIncorrectId = '6460565349c6fa0cfd69db87';
+beforeAll(async () => {
+  await superAdmins.collection.insertMany(superAdminsSeed);
+});
+describe('PUT /api/super-admins', () => {
+  test('Sending same data as in the DB, should return a status 404 and a message', async () => {
+    const response = await request(app).put(`/api/super-admins/${superAdminsSeed[0]._id}`).send(mockSuperAdminSameInfo);
+    expect(response.status).toBe(404);
+    expect(response.body.error).toBeTruthy();
+    expect(response.body.message).toBeDefined();
+    expect(response.body.data).toBeUndefined();
+  });
+  test('No body is send on the request, should return a status 400 and a message', async () => {
+    const response = await request(app).put(`/api/super-admins/${superAdminsSeed[0]._id}`).send();
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBeTruthy();
+    expect(response.body.message).toBeDefined();
+    expect(response.body.data).toBeUndefined();
+  });
+  test('No body and no Id is send on the request, should return a status 400 and a message', async () => {
+    const response = await request(app).put('/api/super-admins/').send();
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBeTruthy();
+    expect(response.body.message).toBeDefined();
+    expect(response.body.data).toBeUndefined();
+  });
+  test('No Id is send on the request, should return a status 400 and a message', async () => {
```

Comment:
I think here the message status code number should be 404
```suggestion
  test('No Id is send on the request, should return a status 404 and a message', async () => {
```

Reply:
fixed